src/components/FeaturePanel/Climbing/ClimbingDialogHeader.tsx

```diff
@@ -80,7 +87,7 @@ export const ClimbingDialogHeader = ({ onClose }) => {
               <PhotoLinks>
                 {photoPaths.map((photo, index) => (
                   <PhotoLink
-                    onClick={() => onPhotoChange(photo)}
+                    onClick={() => onPhotoChange(photo, index)}
```

Comment:
Could you use real link `a` with `href`? I think it is better to use links if possible, as user can cmd+click it etc.